src/app/modules/training-program/components/programs/program-list.component.css

```diff
@@ -1 +1,17 @@
- 
\ No newline at end of file
+.loading-shade {
```

Comment:
Remove re-defining these styles, they're already been defined in `src/styles.css`